scripts/coverage.sh

```diff
@@ -1,32 +0,0 @@
-#! /bin/bash
-set -e
-
-GITREPO_ROOT="${1:-$(readlink -f "$(dirname "$0")/..")}"
-
-export SOFTHSM2_CONF="$PWD/build-coverage/softhsm2.conf"
-export TOKEN_DIR="$PWD/build-coverage/softhsm2-tokens"
-
-mkdir -p build-coverage
-rm -rf "${TOKEN_DIR}"
-mkdir -p "${TOKEN_DIR}"
-cp "$GITREPO_ROOT/tests/test_data/softhsm2.conf" "${SOFTHSM2_CONF}"
-
-"$GITREPO_ROOT/scripts/setup_hsm.sh"
-
-(
-cd build-coverage
-cmake -DBUILD_OSTREE=ON -DBUILD_SOTA_TOOLS=ON -DBUILD_DEB=ON -DBUILD_P11=ON -DBUILD_WITH_CODE_COVERAGE=ON -DTEST_PKCS11_ENGINE_PATH="/usr/lib/engines/engine_pkcs11.so" -DTEST_PKCS11_MODULE_PATH="/usr/lib/softhsm/libsofthsm2.so" "${GITREPO_ROOT}"
-make -j8
-if [ "$JENKINS_RUN" = 1 ]; then
-    echo "Run as jenkins"
-    CTEST_OUTPUT_ON_FAILURE=1 CTEST_PARALLEL_LEVEL=3 make -j6 coverage || true
-else
-    CTEST_OUTPUT_ON_FAILURE=1 CTEST_PARALLEL_LEVEL=3 make -j6 coverage
-fi
-)
-
-if [ -n "$TRAVIS_COMMIT" ]; then
-    bash <(curl -s https://codecov.io/bash -p src -s build-coverage)
```

Comment:
The options here were supplied to curl :) and our `codecov.yml` was never actually read by the tool (not on recent PRs at least)